site-manage/site-manage-tool/tool/src/bundle/sitesetupgeneric.properties

```diff
@@ -222,7 +222,7 @@ import.links.warning = WARNING: If you do not select to import ALL materials fro
 import.options.title = Options
 import.options.help = This tool has import options. Click to see them.
 import.options.publish = Publish content
-import.options.publish.help = Selecting this will publish the tool content and will also import all the associated items. Those may be gradebook items, calendar events, etc.
+import.options.publish.help = Selecting this will publish the tool content and will also import all the associated items. Those may be gradebook items, calendar events, etc. Assignments using groups will be imported as a draft even if you elect to publish them here.
```

Comment:
Assignments released to groups will always be imported as drafts as they require manual group setup before publishing.